reference/pipeline/expressions/index.md

```diff
@@ -0,0 +1,207 @@
+
+Retrieves the contents of the given URL and converts it into either a map or a
+list. You can use this to fetch information from the Spinnaker API or other
+sources.
+
+### #judgment(String)
+
+Returns the selected judgment value from the Manual Judgment stage whose name
+matches the input string. Note that `#judgment` is case sensitive:
+`${#judgment("my manual judgment stage")}` returns an error if your stage is
+named _"My Manual Judgment Stage"_.
+
+### #propertiesFromUrl(String)
+
+Retrieves the contents of the properties file at the given URL and converts it
+into a map. You can use this to fetch information from Jenkins properties files
+or other similar endpoints.
+
+### #stage(String)
+
+A shortcut to get the stage by name. For example, `${#stage("Bake")}` allows
+you to access your Bake stage. Note that `#stage` is case sensitive: if your
+stage is actually named "_bake_", `${#stage("Bake")}` will not find it. Remember
+that the values for the stage are still under the context, so you would access a
+property via `${#stage("Bake")["context"]["desiredProperty"]}`.
+
+### #toBoolean(String)
+
+Converts the input string to a boolean.
+
+### #toFloat(String)
+
+Converts a value to a floating point number.
+
+### #toInt(String)
+
+Converts a value to an integer.
+
+### #toJson(Object)
+
+Converts a map into a JSON string.
+
+## Whitelisted Java classes
+
+You can find the code which whitelists Java classes [here](https://github.com/spinnaker/orca/blob/6d0ba0bf8af5e06c5b405b8294f07e7a5a4c335a/orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/expressions/whitelisting/InstantiationTypeRestrictor.java#L26).
+The whitelisted classes are:
+
+* [Boolean](https://docs.oracle.com/javase/8/docs/api/java/lang/Boolean.html)
+* [Byte](https://docs.oracle.com/javase/8/docs/api/java/lang/Byte.html)
+* [Date](https://docs.oracle.com/javase/8/docs/api/java/util/Date.html)
+* [Double](https://docs.oracle.com/javase/8/docs/api/java/lang/Double.html)
+* [Integer](https://docs.oracle.com/javase/8/docs/api/java/lang/Integer.html)
+* [LocalDate](https://docs.oracle.com/javase/8/docs/api/java/time/LocalDate.html)
+* [Long](https://docs.oracle.com/javase/8/docs/api/java/lang/Long.html)
+* [Math](https://docs.oracle.com/javase/8/docs/api/java/lang/Math.html)
+* [Random](https://docs.oracle.com/javase/8/docs/api/java/util/Random.html)
+* [SimpleDateFormat](https://docs.oracle.com/javase/8/docs/api/java/text/SimpleDateFormat.html)
+* [String](https://docs.oracle.com/javase/8/docs/api/java/lang/String.html)
+* [UUID](https://docs.oracle.com/javase/8/docs/api/java/util/UUID.html)
+
+## Source code
+
+Source code for the expression language is in the [spinnaker/orca
+repository](https://github.com/spinnaker/orca), mostly in the
+[ContextParameterProcessor class](https://github.com/spinnaker/orca/blob/master/orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/util/ContextParameterProcessor.java).
+
+
+## Pipeline expression language reference
```

Comment:
I feel like "pipeline expression language reference" is what we're reading already. Is there some other way you can refer to the SpEL reference?